test/exprtest/blas1_iface_test.cpp

```diff
@@ -25,6 +25,8 @@
 
 #include "blas_test.hpp"
 
+// This test is not build or executed, and will not compile
```

Comment:
(buil**t**)
What is this test for then? Should this stay?

Reply:
I was updating this test, and saw that it wasn't being built, so figured I'd add a note to future travellers. Since Przemek seems to have removed it, I'll remove my change for a cleaner merge.